index.html

```diff
@@ -22,6 +59,7 @@
   </noscript>
 
   <div id="app"></div>
+  <script type="module" src="/src/main.ts"></script>
```

Comment:
**Question**: Any particular reason to move this? Have you compared the output between both versions?

Reply:
So that the loader is loaded/parsed before we make this request also see https://github.com/kumahq/kuma-gui/pull/1610#discussion_r1363558356

Reply:
P.S. 

> Have you compared the output between both versions?

I have tried it out yes, if thats what you mean? FYI: Ideally I would like the kuma gif to be inlined also, but I didn't want to straight inline it into the file. I was wondering if there was a vite plugin to do that for me, whichever we can add this at a later date.